client/my-sites/themes/theme-showcase.jsx

```diff
@@ -795,7 +794,6 @@ const mapStateToProps = ( state, { siteId, filter } ) => {
 		isSiteWooExpress: isSiteOnWooExpress( state, siteId ),
 		isSiteWooExpressOrEcomFreeTrial:
 			isSiteOnECommerceTrial( state, siteId ) || isSiteOnWooExpress( state, siteId ),
-		isSearchV2: ! isUserLoggedIn( state ) && config.isEnabled( 'themes/text-search-lots' ),
```

Comment:
The feature flag "config.isEnabled( 'themes/text-search-lots' )" is ON at each environment, so I cleared it. No need anymore.